spyder/plugins/editor/widgets/tests/assets/autopep8_range.py

```diff
@@ -10,13 +10,11 @@
 
 # %% functions
 def d():
-    def inner():
-        return 2
-
+    def inner(): return 2
     return inner
+# ---- func 1 and 2
 
 
```

Comment:
```suggestion
    return inner


# ---- func 1 and 2
```

Reply:
I can't apply this suggestion and the ones below because they are the output expected by autopep8.